### DISCUSSION TITLE
Results of Training Based on the Recent - 11/6/2022 Updates

### DISCUSSION THREAD

**Discussion by MasterDenis** [TheLastBen/fast-stable-diffusion#414] [General]
Okay so I have conducted a lot of sessions, in the expense of GPU Credits I got the following:

IMPORTANT (May Help to Fine-Tune):
I did another experiment and I confirm that my most stable results were 3000 steps and 31.7% text encoder with 20 images (and **class images at 175**).
I tampered with the num_class_images because from what I have read 200 is too high and 50 is way too low, so 175 with strictly 20 images would produce ALMOST perfect 1:1 likeness at CFG 3.5 and CFG 4 with your prompt.
Problem is anything above that will collapse and distort (wrong features and such).

(I cannot show you the actual pictures because its a disclosed subject but I can describe my results)
**Input**:
**Images**: 20
**Unet Steps**: 3000
**Text Encoder** 31.7% (951 txt_encoder steps)
**Num Class Images**: 175

**Testing Results**:

CFG:
--->3.0, 3.5, 4 = **Perfect 1:1 Likeness** (Almost Couldn't tell if its real or not)
4.5 = Distorted Features, very harsh pimples represented as harsh lines.
5.0, 5.5 = Same as 4.5 but worse eyes and such.
6 = Same as 5.0 but way worse face features, very distorted.
6.5, 7.0 = started to calm down the distortion BUT the pimples are shown as lines that connect with each other almost, other features are correct aside from eyes.
7.5 = Almost Correct features with some distortion of the pimples as described.
8.0 + is distorted and loses the features bit by bit as I increase the scale.

Prompt Used: Token, Still frame, cinematic, 1970s film, 40mm f/2.8, 4k uhd
Negative Prompt Used: cartoon, fake, painting
Sampler: Euler A,
Steps: 35
(Both 1 and 4 Batch Size Compared)
Seed: 3240882916
->I tried with other seeds, similar results.

I hope that helps! @TheLastBen

**Question**:
I would like to get the same results as I get in 3.0 with the above settings but at higher scales so I can do prompt engineering better, any ideas?

**Comment by MasterDenis** [comment]
PS: The above settings make it hard to stylize due to the cfg being too low so the program will do more or less what it wants.

**Reply by d1mski** [reply]
Very interesting. I've been getting very inconsistent results. Training with 22 images at 4400 steps with text_encoder at 22% as well as 35% produces accurate results but very difficult to stylize. I have also encountered artifacts and pimples on the face as described elsewhere. Before the update (when the fp16 checkbox was available), I was getting very consistent results with 1500 to 2000 steps and 14-16 images.

**Reply by TheLastBen** [reply]
You can change "fp16" to "no" in the code cell, for better quality, use 640x640 pictures or more, example of training on 640x640 (fp16 on) (photography style) : 

![tmpujb7twwj](https://user-images.githubusercontent.com/110583491/200930572-9c3687d2-8273-41bb-9010-e21d86e4c2c1.png)

![download (26)](https://user-images.githubusercontent.com/110583491/200930641-f0ac1bf4-162a-4b2d-9829-d2c450333442.png)

![download (7)](https://user-images.githubusercontent.com/110583491/200930708-50a6d1cd-566f-4a63-97a0-c8fe6e5a4d60.png)

![download (3)](https://user-images.githubusercontent.com/110583491/200930755-a787251c-32c7-40b5-a1e6-616fc4c4a906.png)

![download (14)](https://user-images.githubusercontent.com/110583491/200930826-725dd2a1-8118-4a06-9a9a-d22f3fd7dc53.png)
